samples/simple-jetty-webapp/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
For XML I use this setting:

![image](https://cloud.githubusercontent.com/assets/99080/19068482/1afb1fe4-8a23-11e6-89c7-f1e9806220e0.png)